### PR TITLE
Add in-toto proposed project

### PR DESCRIPTION
**Pull request by JustinCappos**

### CODE REVIEW

**Comment by nate-double-u** [comment]
Should https://github.com/in-toto/docs/issues/85 be the upstream issue here?

File: programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md [hunk 1]
```diff
@@ -188,3 +188,24 @@ Onboarding new end users into a sophisticated system like Knative Eventing prese
...
+  - Develop effective documentation for CNCF projects.  As a start, the CNCF project in-toto has a fairly clear set of requirements for what
+documentation changes are needed.
+- Recommended Skills:
+  - Technical writing
+  - Basic understanding of cloud native projects (or a desire to learn!)
+- Mentor(s):
+  - Justin Cappos @JustinCappos jcappos@nyu.edu
+  - LF documentation wizard (to be named later)
+- Upstream Issues:
+  -   https://github.com/in-toto/docs/issues/90
```
Comment (JustinCappos) [inline programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md:209]:
```suggestion
  -   https://github.com/in-toto/docs/issues/85
  -   https://github.com/in-toto/docs/issues/90
```

**Comment by JustinCappos** [comment]
> Should [in-toto/docs#85](https://github.com/in-toto/docs/issues/85) be the upstream issue here?

Thanks, added it!

**Comment by aliok** [comment]
We either wait until there's a 2nd mentor, or we remove that and just go with a single mentor.
We can wait until the EOD Thursday latest.

File: programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md [hunk 2]
```diff
@@ -188,3 +188,25 @@ Onboarding new end users into a sophisticated system like Knative Eventing prese
...
+point to help start your career in open source.
+- Expected Outcome:
+  - Develop effective documentation for CNCF projects.  As a start, the CNCF project in-toto has a fairly clear set of requirements for what
+documentation changes are needed.
+- Recommended Skills:
+  - Technical writing
+  - Basic understanding of cloud native projects (or a desire to learn!)
+- Mentor(s):
+  - Justin Cappos @JustinCappos jcappos@nyu.edu
+  - LF documentation wizard (to be named later)
```
Comment (chalin) [inline programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md:207]:
```suggestion
  - Patrice Chalin @chalin chalin@cncf.io
```

**Review by chalin — COMMENTED** [review]
Hi @JustinCappos & @aliok - I'll be joining you as second mentor 👋 

/cc @caniszczyk @krook @nate-double-u

**Comment by aliok** [comment]
Thanks, merging this one now!

**Review by aliok — APPROVED** [review]

**Pull request merged**